src/commands/versions.ts

```diff
@@ -0,0 +1,68 @@
+import { iCommand } from './commands';
+import { red, blue } from 'colors';
+import { emoji } from 'node-emoji';
+import { Utils } from '../utils/utils';
+
+export default class Versions implements iCommand {
+  private logo = `
+                            *//                            
+                        ,****/////,                        
+                     ********/////////                     
+                  *********     /////////                  
+              *********.    *//    ./////////              
+           *********     ****/////     /////////           
+       *********,    ********////////*    */////////       
+   .*********     ***********////////////     /////////.   
+ ********,    ***************///////////////*    *///////* 
+ ,****     **********  ******//////*  //////////     ////. 
+ .****  *********          **///         .////////*  ////  
+  ****  ,*****                   .******     /////.  ////  
+  ****   ****                  ***********/  .////  .////  
+  ****.  ****                  ****/**/////  .////  *////  
+  ****,  ****                  ******//////  .////  /////  
+  *****  ****                  ******//////  .////  /////  
+  *****  ****                   .****////    .////  /////  
+  *****  *******.           ***     *     ,///////  ////*  
+  *****  ***********    ***********    ///////////  ////.  
+  .****  ,***********  **********///  ///////////,  ////   
+   ****  .***********  ******///////  ///////////   ////   
+   ****   ***********  ******///////  ///////////  .////   
+   ****   ***********   *****//////   ///////////  *////   
+   ******    *********,    .*//     *////////*    //////   
+   *********     *********       /////////     ////////,   
+      ,********     *********./////////     ////////.      
+          ********,    ******///////    *////////          
+             *********    ***////    /////////             
+                *********    .    ////////*                
+                   ,********  .////////.                   
+                       ******///////                       
+                          ***////                                                                                 
+                                                                                                      
+`;
+  execute() {
+    Utils.log(this.logo, red);
+    let localPackageJson: any = require(process.cwd() + '/package.json');
+    let cliPackageJson: any = require(__dirname + '/../../package.json');
+
+    if (cliPackageJson && cliPackageJson.version) {
+      Utils.log(emoji.heavy_check_mark + " Covalent CLI Version: " + cliPackageJson.version, blue);
+    }
+    if (localPackageJson && localPackageJson.dependencies) {
+      if (localPackageJson.dependencies['@covalent/core']) {
```

Comment:
So, this will get the version thats written in our package.json.. so potentially it could say `^1.0.0` but the one installed might be `1.1.0`.. so would be nice if we can go to `node_modules` and check the installed version.